kolibri/plugins/coach/assets/src/composables/useFetchContentNode.js

```diff
@@ -0,0 +1,38 @@
+import { ref } from 'vue';
+import ContentNodeResource from 'kolibri-common/apiResources/ContentNodeResource';
+
+export default function useFetchContentNode(contentId) {
+  const contentNode = ref({});
+  const ancestors = ref([]);
+  const questions = ref([]);
+  const loading = ref(true);
+
+  const fetchContentNode = async () => {
+    ContentNodeResource.fetchModel({
+      id: contentId,
+      getParams: { no_available_filtering: true },
+    }).then(node => {
+      contentNode.value = node;
+      if (node != null) {
+        loading.value = false;
+      }
+
+      if (node.ancestors.length) {
+        ancestors.value = node.ancestors;
```

Comment:
If `node` can be true (as I imply because of the previous check), we will have an exception here.